dandiapi/api/tests/factories.py

```diff
@@ -116,7 +151,30 @@ class AssetMetadataFactory(factory.django.DjangoModelFactory):
     class Meta:
         model = AssetMetadata
 
-    metadata = factory.Faker('pydict', value_types=['str', 'float', 'int'])
+    @factory.lazy_attribute
+    def metadata(self):
+        return {
+            'schemaVersion': '0.3.1',
+            'encodingFormat': 'application/x-nwb',
+            'publishedBy': {
+                'id': 'urn:uuid:08fffc59-9f1b-44d6-8e02-6729d266d1b6',
+                'name': 'DANDI publish',
+                'startDate': '2021-05-18T19:58:39.310338-04:00',
+                'endDate': '2021-05-18T19:58:39.310361-04:00',
+                'wasAssociatedWith': [
+                    {
+                        'id': 'RRID:SCR_017571',
+                        'name': 'DANDI API',
+                        'version': '0.1.0',
```

Comment:
this should pick up the version of the API.

Reply:
For now the API is unversioned, but as soon as it is it will be added here.